ci/docker/tizen/build-docker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+IMAGE_NAME=ghcr.io/flutter-tizen/build-engine
+IMAGE_TAG=latest
+GIT_BRANCH=flutter-2.2.1-tizen
```

Comment:
Should we hard-code this branch name?

Where is this file (`build-docker.sh`) used?

Reply:
Actually, this script is not used anywhere, it is just for local testing.

Reply:
If so, can you remove the file or at least replace the hard-coded part with another?

Reply:
okay, I removed the hard-coded part.